index.js

```diff
@@ -933,6 +939,7 @@ const _bindWindow = (window, newWindowCb) => {
 
     // update media frames
     nativeVideo.Video.updateAll();
+    nativeVideo.VideoCapture.updateAll();
```

Comment:
Is this accidentally coming from another PR?

Reply:
Drat, yes.